src/site/content/en/blog/requestvideoframecallback-rvfc/index.md

```diff
@@ -55,22 +55,28 @@ but I'm happy with the new name,
 after a [lengthy discussion](https://github.com/WICG/video-rvfc/issues/44).
 Yay, [bikeshedding](https://css-tricks.com/what-is-bikeshedding/) for the win!
 
-## Browser support and feature detection
-
-The method is
-[implemented in Chromium](https://chromestatus.com/feature/6335927192387584)
-already, and
-[Mozilla folks like it](https://mozilla.github.io/standards-positions/#requestVideoFrameCallback).
-For what it's worth, I have also filed a
-[WebKit bug](https://bugs.webkit.org/show_bug.cgi?id=211945) asking for it.
-Feature detection of the API works like this:
+## Feature detection
 
 ```js
 if ('requestVideoFrameCallback' in HTMLVideoElement.prototype) {
   // The API is supported! 
 }
 ```
 
+## Browser support
+
+{% BrowserCompat 'api.HTMLVideoElement.requestVideoFrameCallback' %}
+
+## Polyfill
+
+A [polyfill for the `requestVideoFrameCallback()` API](https://github.com/ThaUnknown/rvfc-polyfill)
```

Comment:
```suggestion
A [polyfill for the `requestVideoFrameCallback()` method](https://github.com/ThaUnknown/rvfc-polyfill)
```

Reply:
"Use definite, specific, concrete language." (Strunk and White, rule II-16.) 'Method' is more specific than 'API'.